src/js/state/migrations/202008191031_changeZqdClusterIdToHostPort.test.ts

```diff
@@ -17,16 +14,13 @@ test("migrating 202008191031_changeZqdClusterIdToHostPort", () => {
   }
 
   // @ts-ignore
-  for (const {state} of windows) {
+  for (const state of getAllStates(next)) {
+    if (!state.clusters) continue
     expect(state.clusters[newId]).toEqual(newValue)
     expect(Object.keys(state.clusters)).not.toContain(oldId)
   }
 
-  // @ts-ignore
-  const tabsData = windows.flatMap((win) => win.state.tabs.data)
-
-  for (const td of tabsData) {
-    if (!td.current) continue
-    expect(td.current.workspaceId).toBe(newId)
```

Comment:
@mason-fish this was what tipped me off to the need for this PR. I'm guessing the migration file was updated as the result of a  find/replace of clusterId to workspaceId. Since this is an old migration file, I expected this test to fail. But it was never getting to that expect call because of the if clause above it. This state has "search.clusterId" instead of "current.clusterId". It was not being caught because the migration before this moves clusterId to the current slice.

Reply:
I see, good catch!